src/dps-email-poller/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/dpsemailpoller/email/api/DpsEmailController.java

```diff
@@ -21,11 +24,15 @@ public DpsEmailController(EmailService emailService) {
         this.emailService = emailService;
     }
 
-    @PostMapping(value = "/email/processed", produces = MediaType.APPLICATION_JSON_VALUE)
+    @PostMapping(value = "/email/processed/{id}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Successful operation", response = DpsEmailResponse.class),
```

Comment:
200: Ok?

Reply:
it's for documentation but yeah it will return a 200 ok